app/models/user.rb

```diff
@@ -1,6 +1,12 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  
+  enum user_type: [:regular_user, :patron]
+  def admin?
+    user_type == 'admin'
+  end
```

Comment:
1. `admin` should be included in `user_type`then we'll not need this admin? method as well
2. Why is `parton` a user_type? You've created a separate model for Parton? Let me know what's your thinking process here, but only one of the two should be used

Reply:
I designed the Patron user type to act as an admin and have certain controls like editing and deleting a book that normal user will not have